target/linux/kirkwood/base-files/etc/board.d/02_network

```diff
@@ -22,6 +22,9 @@ kirkwood_setup_interfaces()
 	globalscale,sheevaplug|\
 	iom,iconnect-1.1|\
 	iom,ix2-200|\
+	iom,ix4-200d)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
+		;;
 	iptime,nas1|\
```

Comment:
This broke other devices like sheevaplug, iconnect or ix2-200.

Reply:
Oryginal:
```
	cloudengines,pogoe02|\
	cloudengines,pogoplugv4|\
	ctera,c200-v1|\
	globalscale,sheevaplug|\
	iom,iconnect-1.1|\
	iom,ix2-200|\
	iptime,nas1|\
	netgear,readynas-duo-v2|\
	raidsonic,ib-nas62x0|\
	seagate,blackarmor-nas220|\
	seagate,dockstar|\
	seagate,goflexhome|\
	seagate,goflexnet|\
	zyxel,nsa310b|\
	zyxel,nsa310s|\
	zyxel,nsa325)
		ucidef_set_interface_lan "eth0" "dhcp"
		;;
```

So your code should be
```
	cloudengines,pogoe02|\
	cloudengines,pogoplugv4|\
	ctera,c200-v1|\
	globalscale,sheevaplug|\
	iom,iconnect-1.1|\
	iom,ix2-200|\
	iptime,nas1|\
	netgear,readynas-duo-v2|\
	raidsonic,ib-nas62x0|\
	seagate,blackarmor-nas220|\
	seagate,dockstar|\
	seagate,goflexhome|\
	seagate,goflexnet|\
	zyxel,nsa310b|\
	zyxel,nsa310s|\
	zyxel,nsa325)
		ucidef_set_interface_lan "eth0" "dhcp"
		;;
       iom,ix4-200d)
		ucidef_set_interfaces_lan_wan "eth0" "eth1"
		;;
```
Not inside existing statement.

Reply:
No. this is "normal", not inside. 
If you have
```
case $var in
a|\
b)
  echo "1"
  ;;
c)
 echo "2'
 ;;
esac
```

code shows "1" for a or b. but if you change (add) "inside":
```
case $var in
a|\
d)
  echo "0"
  ;;
b)
  echo "1"
  ;;
c)
 echo "2'
 ;;
esac
```

Then for "a" shows 0, not 1, as previous. So you code should not break existing behaviour.